mlir/lib/Dialect/MIOpen/Generator/Conv2dGenerator.cpp

```diff
@@ -36,6 +36,11 @@ Conv2dGenerator::Conv2dGenerator(
              inputLayout,
              outputLayout,
              kernelName,
+             -1,
+             {},
+             {},
+             {},
+             -1,
              -1} {}
```

Comment:
@krzysz00 Your changes LGTM

@ltqin The refactor from @krzysz00 caught my attention about the last two member variables: `filterHeight` and `filterWidth`, first introduced [here](https://github.com/ROCmSoftwarePlatform/llvm-project-mlir/pull/246/files). I don't think it is the right way to store this duplicated information as those should be available from `filterDimension` vector. Let me know if you think differently on this, otherwise please create a PR/ticket and get rid of those two member variables.

Reply:
Hearing no reply, recorded this in a ticket https://github.com/ROCmSoftwarePlatform/llvm-project-private/issues/255